plutus-pab/src/Cardano/Node/Server.hs

```diff
@@ -58,17 +58,18 @@ main :: Trace IO MockServerLogMsg -> MockServerConfig -> Availability -> IO ()
 main trace MockServerConfig { mscBaseUrl
                             , mscRandomTxInterval
                             , mscBlockReaper
```

Comment:
You can delete `mscBlockReaper` and everything connected to it, it was trying to do something similar

Reply:
Yeah. It's on the todo list.